index.html

```diff
@@ -1,21 +1,270 @@
+                        Is Credard a free product ?<img
+                            class="arrow-icon"
+                            src="./assets/Icons/arrow-down.svg"
+                            alt=""
+                        />
+                    </summary>
+                    <p class="faq_reply">Answer</p>
+                </details>
+                <details class="faq_item">
+                    <summary class="faq_question">
+                        Can I only download a JPEG file ?<img
+                            class="arrow-icon"
+                            src="./assets/Icons/arrow-down.svg"
+                            alt=""
+                        />
+                    </summary>
+                    <p class="faq_reply">Answer</p>
+                </details>
+                <details class="faq_item">
+                    <summary class="faq_question">
+                        Is it possible to save my designs ?<img
+                            class="arrow-icon"
+                            src="./assets/Icons/arrow-down.svg"
+                            alt=""
+                        />
+                    </summary>
+                    <p class="faq_reply">Answer</p>
+                </details>
+            </section>
+        </main>
+        <footer class="footer">
+            <div class="footer_top">
+                <div>
+                    <img src="./assets/Icons/Group 44.svg" alt="Logo" />
+                </div>
+
+                <div class="footer_links">
+                    <nav>
+                        <h3 class="footer_heading">Support</h3>
+                        <ul class="footer_list">
+                            <li>
+                                <a class="footer_link" href="#">Contact us</a>
+                            </li>
+                        </ul>
+                    </nav>
+                    <nav>
+                        <h3 class="footer_heading">Library</h3>
+                        <ul class="footer_list">
+                            <li>
+                                <a class="footer_link" href="#">View Library</a>
+                            </li>
+                            <li>
+                                <a class="footer_link" href="#"
+                                    >Saved Designs</a
+                                >
+                            </li>
+                        </ul>
+                    </nav>
+                    <nav>
+                        <h3 class="footer_heading">Company</h3>
+                        <ul class="footer_list">
+                            <li>
+                                <a class="footer_link" href="#"
+                                    >Meet the Team</a
+                                >
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+            <hr class="footer_hr" />
```

Comment:
Include the footer in the global branch to this one, so we won't have to do it again, thanks.